programs/eosio-launcher/main.cpp

```diff
@@ -1635,6 +1654,11 @@ launcher_def::kill (launch_modes mode, string sig_opt) {
   case LM_LOCAL:
   case LM_REMOTE : {
     bfs::path source = "last_run.json";
+    if (cluster_id) {
+      std::stringstream s;
```

Comment:
Similar change as before: use string concatenation instead and enforce 5 digits, e.g. `last_run_cluster00001.json`.